src/core/trulens/core/schema/feedback.py

```diff
@@ -192,6 +202,9 @@ class FeedbackResult(serial.SerialModel):
         default_factory=datetime.datetime.now
     )
 
+    run_location: Optional[FeedbackRunLocation]
```

Comment:
That way we don't have to duplicate it in every row.

Reply:
Yeah initially I did it this way since I wouldn't have to join the feedback definition table, but I guess that's not a big deal so I went ahead with your suggested approach. PTAL.